js/index.js

```diff
@@ -37,6 +37,71 @@ const siteContent = {
   },
 };
 
+///////////////////    NAV   /////////////////
+///////////////////    IMAGE   /////////////////
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+///////////////////    ANCHORS   /////////////////
+let anchor = document.querySelectorAll('a');
+anchor[0].textContent = siteContent['nav']['nav-item-1'];
+anchor[1].textContent = siteContent['nav']['nav-item-2'];
+anchor[2].textContent = siteContent['nav']['nav-item-3'];
+anchor[3].textContent = siteContent['nav']['nav-item-4'];
+anchor[4].textContent = siteContent['nav']['nav-item-5'];
+anchor[5].textContent = siteContent['nav']['nav-item-6'];
+anchor.forEach(element => {element.style.color = 'green'});
+///////////////////    NEW TAGS   /////////////////
+const navBar = document.querySelector('nav');
+let newAnchorTag1 = document.createElement('a');
+    newAnchorTag1.textContent = 'Lightbulb';
+    newAnchorTag1.style.color = 'green';
+    navBar.appendChild(newAnchorTag1);
+let newAnchorTag2 = document.createElement('a');
+    newAnchorTag2.textContent = 'Home';
+    newAnchorTag2.style.color = 'green';
+    navBar.prepend(newAnchorTag2);
+
+ ///////////////////    HEADER/CTA   /////////////////
+///////////////////    TITLE   /////////////////
+let titleDom = document.querySelector('h1');
+titleDom.innerHTML = 'DOM<br>is<br> Awesome';
+///////////////////    BUTTON   /////////////////
+let btn = document.querySelector('button');
+btn.textContent = siteContent['cta']['button'];
+///////////////////    IMAGE   /////////////////
+let logoCircle = document.getElementById("cta-img");
+logoCircle.setAttribute('src', siteContent.cta["img-src"])
+
+
+///////////////////    MAIN-CONTENT   /////////////////
+///////////////////    IMAGE   /////////////////
+let midImage = document.getElementById("middle-img");
```

Comment:
You can actually make this a one liner if you want like this : `let midImage = document.getElementById('middle-img').src = siteContent['main-content']['middle-img-src'];`